src/Models/AdminUser.php

```diff
@@ -3,17 +3,24 @@
 use Brackets\AdminAuth\Activation\Contracts\CanActivate as CanActivateContract;
 use Brackets\AdminAuth\Activation\Traits\CanActivate;
 use Brackets\AdminAuth\Notifications\ResetPassword;
+use Brackets\Media\HasMedia\HasMediaCollections;
```

Comment:
This package does not require media package.

Reply:
Also I have tested this, but it will not save the avatar image to my profile, not sure why.